packages/medusa-plugin-restock-notification/package.json

```diff
@@ -38,5 +38,9 @@
     "express": "^4.17.1",
     "medusa-core-utils": "^1.1.31"
   },
-  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e"
+  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-notification"
```

Comment:
todo: Not a notification plugin, but hard to know given the naming 😅 It's just a custom feature for sending out restock emails

Reply:
Should we just leave it under "other" then?